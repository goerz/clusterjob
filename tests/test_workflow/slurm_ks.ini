[Attributes]
remote = kcluster
backend = slurm
shell = /bin/bash

[Resources]

queue = exec
nodes = 1
threads = 1
mem = 20
