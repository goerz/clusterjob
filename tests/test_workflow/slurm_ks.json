[
  {
    "args": [
      [
        "mkdir",
        "-p",
        "./."
      ],
      "kcluster"
    ],
    "kwargs": {
      "ignore_exit_code": false,
      "ssh": "ssh"
    },
    "response": ""
  },
  {
    "args": [
      [
        "sbatch",
        "test_clj.slr"
      ],
      "kcluster",
      ".",
      "."
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "ssh"
    },
    "response": "Submitted batch job 5460093\n"
  },
  {
    "args": [
      [
        "squeue",
        "-h",
        "-o %T",
        "-j 5460093"
      ],
      "kcluster"
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "ssh"
    },
    "response": "RUNNING\n"
  },
  {
    "args": [
      [
        "scancel",
        "5460093"
      ],
      "kcluster"
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "ssh"
    },
    "response": ""
  },
  {
    "args": [
      [
        "mkdir",
        "-p",
        "./."
      ],
      "kcluster"
    ],
    "kwargs": {
      "ignore_exit_code": false,
      "ssh": "ssh"
    },
    "response": ""
  },
  {
    "args": [
      [
        "sbatch",
        "test_clj.slr"
      ],
      "kcluster",
      ".",
      "."
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "ssh"
    },
    "response": "Submitted batch job 5460094\n"
  },
  {
    "args": [
      [
        "squeue",
        "-h",
        "-o %T",
        "-j 5460094"
      ],
      "kcluster"
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "ssh"
    },
    "response": "RUNNING\n"
  },
  {
    "args": [
      [
        "squeue",
        "-h",
        "-o %T",
        "-j 5460094"
      ],
      "kcluster"
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "ssh"
    },
    "response": ""
  },
  {
    "args": [
      [
        "sacct",
        "--format=state",
        "-n",
        "-j 5460094"
      ],
      "kcluster"
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "ssh"
    },
    "response": " COMPLETED \n COMPLETED \n"
  }
]