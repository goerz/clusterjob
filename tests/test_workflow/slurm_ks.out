slurmd[its-cs325]: Unable to get current working directory: No such file or directory
####################################################
Job id         : 5460094
Job name       : test_clj
Workdir        : /gpfs/home08/0001/uk006169
Submission Host: its-cs1.its.uni-kassel.de
Compute Node   : its-cs325
Job started on its-cs325.its.uni-kassel.de Thu Feb 25 22:33:59 CET 2016
Current directory: /gpfs/home08/0001/uk006169
####################################################
Job Finished:  Thu Feb 25 22:36:59 CET 2016
